src/index.html

```diff
@@ -8,10 +8,174 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+    <fieldset class="fieldset">
+      <legend>
+        Personal information
+      </legend>
+
+      <label for="surname" class="label">
+        Surname:
+        <input
+        id="surname"
+        type="text"
+        autocomplete="off"
+        required
+        >
```

Comment:
fix attributes indents everywhere
```suggestion
        <input
          id="surname"
          type="text"
          autocomplete="off"
          required
        >
```

Reply:
corrected